src/main/java/com/denizenscript/clientizen/mixin/KeyBindingMixin.java

```diff
@@ -17,10 +17,10 @@ public class KeyBindingMixin {
 
 	@Inject(method = "setKeyPressed", at = @At("HEAD"))
 	private static void clientizen$onKeyPressStateChanged(InputUtil.Key key, boolean pressed, CallbackInfo ci) {
-		if (KeyPressReleaseServerEvent.instance.enabled && key.getCategory() == InputUtil.Type.KEYSYM) {
+		if (KeyPressReleaseEvent.instance != null && key.getCategory() == InputUtil.Type.KEYSYM) {
```

Comment:
Miiiight be wrong, but don't think it'd ever be null? Denizen-Core creates a new instance when registering it, which means the `instance` field will have a value pretty much immediately